documentation/commands/openapi.md

```diff
@@ -257,3 +258,39 @@ EXAMPLES
 
     $ rdme openapi:validate
 ```
+
+## `rdme openapi:refs [SPEC]`
```

Comment:
this document is auto-generated from the code — you can regenerate this by running `npm run build && npm run build:docs`. can you do that instead? the docs you wrote here are great and we should definitely use them! you should be able to supply them via the `summary`, `description`, and `example` methods on your main command class.